.github/workflows/postman-collection-runner.yml

```diff
@@ -3,8 +3,8 @@ name: Run postman tests
 on:
   workflow_dispatch:
   pull_request:
```

Comment:
do you want to remove even `pull_request:` ?

Reply:
Since we're just skipping it on PRs, I don't find the need to disable it here, it will pass here no matter what